### DISCUSSION TITLE
Test for back_url: failure

### DISCUSSION THREAD

**Discussion by caco13** [mercadopago/sdk-js#105] [General]
Hi, I'm trying to test for back_url for the failure case, like in

```
'back_urls': {
                'success': 'www.example.com',
                'failure': 'www.example.com',
                'pending': 'www.example.com',
},
```

Aside to reproduce manually using Checkout Pro, that I also would like how to do, I'd like to know if the payment is created in MercadoPago and so I can recover the data from Mercado Pago via API.

Or else, nothing is created in Mercado Pago?

I. e., the failure is related to the payment not have been created in Mercado Pago?

**Comment by icaldana** [comment]
Hi @caco13! 😺 

We apologize for the inconvenience in your workflow.
Even though this space is open, it is not intended to be a tool to support unexpected errors of the platform and its integrations.

We have an official channel prepared to solve your case quickly.

So if the error still persists, please pile up all the information and access the following link to share the problem with us: https://www.mercadopago.com.br/developers/pt/support/center/tickets

Thank you so much for your contact!
